rest/rest-cli/src/main/java/org/ow2/proactive_grid_cloud_portal/cli/cmd/rm/CreateNodeSourceCommand.java

```diff
@@ -42,6 +42,11 @@
 import org.ow2.proactive_grid_cloud_portal.cli.utils.QueryStringBuilder;
 
 
+/**
+ * @deprecated  As of release 7.37, replaced by
+ * {@link DefineNodeSourceCommand)} and {@link DeployNodeSourceCommand}
+ */
+@Deprecated
 public class CreateNodeSourceCommand extends AbstractCommand implements Command {
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1133)